WcaOnRails/app/webpacker/lib/wca-live/attempts.js

```diff
@@ -81,11 +227,147 @@ export function formatAttemptResult(attemptResult, eventId) {
   if (attemptResult === SKIPPED_VALUE) return '';
   if (attemptResult === DNF_VALUE) return 'DNF';
   if (attemptResult === DNS_VALUE) return 'DNS';
-  if (eventId === '333mbf' || eventId === '333mbo') return formatMbldAttemptResult(attemptResult);
+  if (eventId === '333mbf') return formatMbldAttemptResult(attemptResult);
   if (eventId === '333fm') return formatFmAttemptResult(attemptResult);
   return centisecondsToClockFormat(attemptResult);
 }
 
+/**
+ * Alters the given MBLD decoded value, so that it conforms to the WCA regulations.
+ */
+export function autocompleteMbldDecodedValue({
+  attempted,
+  solved,
+  centiseconds,
+}) {
+  if (!attempted || solved > attempted) {
+    return { solved, attempted: solved, centiseconds };
+  }
+  // See https://www.worldcubeassociation.org/regulations/#9f12c
+  if (solved < attempted / 2 || solved <= 1) {
+    return { solved: 0, attempted: 0, centiseconds: DNF_VALUE };
+  }
+  // See https://www.worldcubeassociation.org/regulations/#H1b
+  // But allow additional (arbitrary) 30 seconds over the limit for possible +2s.
```

Comment:
Nitpick: We _could_ calculate the additional +2 threshold by doing something like `2 * 100 * N * attempted` where `N` is an estimate of the number of +2s you can get per cube. Do you think this would be better than the arbitrary 30 second backoff?

Reply:
I've actually removed from that file all the functions we are not using at the moment!